Game/Source/GDKTestGyms/Public/NFRConstants.h

```diff
@@ -0,0 +1,26 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogNFRConstants, Log, All);
+
+class NFRConstants
+{
+	int64 TimeToStartFPSSampling;
+	bool bFPSSamplingValid{false};
+	bool bInitialized{ false };
+	float MinServerFPS = 20.0f;
+	float MinClientFPS = 20.0f;
+
+	NFRConstants() {}
+public:
+	
+	bool SamplesForFPSValid(); // Wait until this is true until checking FPS values
```

Comment:
uber nit: can we put public section at the top, order the functions according to use (singleton getter first) etc. then add the private section for methods and properties.